### ISSUE TITLE
Local development

### ISSUE THREAD

**Issue by retrodans** [OxDUG/issue-tracker#2]
So everyone can get involved, we need to get a VM setup really so we can stick it on a USB pen for all to grab.  I generally use VLAD, but I know others use Parrot and others.  Also, maybe the hosting we use will come with a VM (do Acquia/Pantheon do this for free hosting?)

Thoughts?

**Comment by darthsteven** [comment]
Does VLAD support Drupal 8 nicely?

Parrot sort of does, but only in a dev branch at the moment.

**Comment by retrodans** [comment]
They are doing  lot of work on getting it to support D8 better I think.  But at the moment, I have VLAD booting up, and a D8 site running inside of it.  I suppose it depends what functionality you are concerned not being there.  If there is something in particular, let me know and I will check.

At the moment I have everything in this working: https://www.haikudeck.com/vlad--xdebug--drupal-8--webprofiler-uncategorized-presentation-1URyo2Wq3t

**Comment by darthsteven** [comment]
Well...nothing huge I guess, but can you run composer in it nicely, can you run Drush 8.

**Comment by retrodans** [comment]
I am not used to composer in all it's glory, but it appears to run

```
$ composer install
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
> Drupal\Core\Composer\Composer::preAutoloadDump
> Drupal\Core\Composer\Composer::ensureHtaccess
```

As for drush 8, is it needed, what with Drupal console handling a lot of the work?

**Comment by darthsteven** [comment]
Maybe not then :)

**Comment by finnlewis** [comment]
Hi guys, sorry for radio silence, I'm keen to do more inter-meetup, but Christmas kinda got in the way of that. At Agile Collective we use Parrot, on a forked feature branch (https://github.com/agilecollective/parrot/tree/feature/1404), and it works fine for us. I've yet to try the https://github.com/computerminds/parrot/tree/trusty branch. And I've not tried VLAD.
I guess there may be more experience with Parrot across Agile Collective and Steven, but always happy to try something new (other than my lack of disk space).

Did either of you have any more thoughts since the discussions above?

**Comment by darthsteven** [comment]
Tempted to say that we should just use Parrot as there's quite a few of us who use that, and we should just get it working.

**Comment by retrodans** [comment]
I agree that with time requiring asap, and also that having more people knowing it will be beneficial.  Whilst I am happy to show people VLAD (and still could in future if people want) having just one person to help fix and debug may not be ideal.

Also, they are likely doing nearly the same thing except one uses puppet and the other ansible. I know VLAD is very configurable and has work constantly being done to it as well if that is a point worth considering.
